README.rst

```diff
@@ -34,6 +36,7 @@ If you use this software for a publication, please cite the proper version using
 
 - v0.7.0 : |doiv07|
 - v0.8.0 : |doiv08|
+- v0.9.1 : |doiv09|
```

Comment:
Small change, but maybe put the newest version on top? Sorting anti-chronologically?